appeals/web/src/server/appeals/appeal-details/development-description/__tests__/development-description.test.js

```diff
@@ -0,0 +1,103 @@
+import { parseHtml } from '@pins/platform';
+import supertest from 'supertest';
+import {
+	appealData,
+	appellantCaseDataIncompleteOutcome
+} from '#testing/app/fixtures/referencedata.js';
+import { createTestEnvironment } from '#testing/index.js';
+import nock from 'nock';
+
+const { app, installMockApi, teardown } = createTestEnvironment();
+const request = supertest(app);
+const baseUrl = `/appeals-service/appeal-details/${appealData.appealId}/appellant-case`;
+
+describe('development-description', () => {
+	beforeEach(installMockApi);
+	afterEach(teardown);
+
+	describe('/GET /change', () => {
```

Comment:
Very minor, but the test names shouldn't have `/` before the method (eg. `GET /change` not `/GET /change`)